app/src/main/kotlin/org/fossify/filemanager/fragments/RecentsFragment.kt

```diff
@@ -196,6 +199,17 @@ class RecentsFragment(context: Context, attributeSet: AttributeSet) : MyViewPage
         }
     }
 
+    private fun isPathInHiddenFolder(path: String): Boolean {
+        val parts = path.split("/")
+        for (i in 1 until parts.size - 1) {
+            val segment = parts[i]
+            if (segment.startsWith(".") && segment != "." && segment != ".." && segment.isNotEmpty()) {
```

Comment:
Use a variable for the condition to fix the detekt check.

```suggestion
            val part = parts[i]
			val isHidden = part.startsWith(".") && part != "." && part != ".." && part.isNotEmpty()
            if (isHidden) {
```